boards/nano_rp2040_connect/Makefile

```diff
@@ -0,0 +1,55 @@
+# Makefile for building the tock kernel for the Arduino Nano RP2040 Connect board.
+
+TOCK_ARCH=cortex-m0p
+TARGET=thumbv6m-none-eabi
+PLATFORM=nano_rp2040_connect
+
+include ../Makefile.common
+
+OPENOCD=openocd
+OPENOCD_OPTIONS=-f openocd.cfg
+
+KERNEL=$(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+KERNEL_WITH_APP=$(TOCK_ROOT_DIRECTORY)/target/$(TARGET)/release/$(PLATFORM)-app.elf
+
+BOOTSEL_FOLDER=/media/$(USER)/RP2040
```

Comment:
I suppose we don't want to define this here? It's better to just show the user the message that `BOOTSEL_FOLDER` is not set than to assume a path and error out.

Reply:
This should be the default for Ubuntu, I think that for more inexperienced users this might be useful.

Reply:
Okay. We should at least make this an overrideable default then (i.e. use `?=`).

Reply:
Done